steampipecloud/table_steampipecloud_connection.go

```diff
@@ -15,10 +12,10 @@ import (
 
 //// TABLE DEFINITION
 
-func tableSteampipecloudConnection(_ context.Context) *plugin.Table {
+func tableSteampipeCloudConnection(_ context.Context) *plugin.Table {
```

Comment:
I'm not sure if it's an issue with my token, local env, or the SDK, but for this table, the `get` call doesn't seem to work when using the `STEAMPIPE_CLOUD_TOKEN` environment variable instead of the `token` argument (in the last query, no results are returned when I'd expect one):

Query with `token` argument set:
```
cbruno@M1P steampipecloud % steampipe query
Welcome to Steampipe v0.10.0
For more information, type .help
> select * from steampipecloud_connection
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
| id                     | identity_id            | identity_handle | identity_display_name | identity_type          | handle | plugin | created_at           | type       | version_id | updated_at | config
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
| c_c6pqf9gaa6gutha5g4ig | u_c6fdkv9e4mvf26h42310 | cbruno10        | cbruno10              | u_c6fdkv9e4mvf26h42310 | github | github | 2021-12-10T19:16:54Z | connection | 1          | <null>     | <null>
| c_c6pqe28aa6gutha5g4c0 | o_c6pqdjoaa6gutha5g3tg | cody-org        | cody-org              | o_c6pqdjoaa6gutha5g3tg | aws    | aws    | 2021-12-10T19:14:17Z | connection | 1          | <null>     | {"access_key":"AKIA4B4LZUOLT4OO4N45","r
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------

> select * from steampipecloud_connection where identity_handle = 'cody-org'
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
| id                     | identity_id            | identity_handle | identity_display_name | identity_type          | handle | plugin | created_at           | type       | version_id | updated_at | config
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
| c_c6pqe28aa6gutha5g4c0 | o_c6pqdjoaa6gutha5g3tg | cody-org        | cody-org              | o_c6pqdjoaa6gutha5g3tg | aws    | aws    | 2021-12-10T19:14:17Z | connection | 1          | <null>     | {"access_key":"AKIA4B4LZUOLT4OO4N45","r
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------

> select * from steampipecloud_connection where identity_handle = 'cody-org' and handle = 'aws';
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
| id                     | identity_id            | identity_handle | identity_display_name | identity_type          | handle | plugin | created_at           | type       | version_id | updated_at | config
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
| c_c6pqe28aa6gutha5g4c0 | o_c6pqdjoaa6gutha5g3tg | cody-org        | cody-org              | o_c6pqdjoaa6gutha5g3tg | aws    | aws    | 2021-12-10T19:14:17Z | connection | 1          | <null>     | {"access_key":"AKIA4B4LZUOLT4OO4N45","r
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
```

Query using `STEAMPIPE_CLOUD_TOKEN` env variable and no config arg set:
```
cbruno@M1P steampipecloud % STEAMPIPE_CLOUD_TOKEN='spt_c6pq...' steampipe query
Welcome to Steampipe v0.10.0
For more information, type .help
> select * from steampipecloud_connection where identity_handle = 'cody-org' and handle = 'aws';
+----+-------------+-----------------+-----------------------+---------------+--------+--------+------------+------+------------+------------+--------+----------+
| id | identity_id | identity_handle | identity_display_name | identity_type | handle | plugin | created_at | type | version_id | updated_at | config | identity |
+----+-------------+-----------------+-----------------------+---------------+--------+--------+------------+------+------------+------------+--------+----------+
+----+-------------+-----------------+-----------------------+---------------+--------+--------+------------+------+------------+------------+--------+----------+

> select * from steampipecloud_connection
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
| id                     | identity_id            | identity_handle | identity_display_name | identity_type          | handle | plugin | created_at           | type       | version_id | updated_at | config
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
| c_c6pqf9gaa6gutha5g4ig | u_c6fdkv9e4mvf26h42310 | cbruno10        | cbruno10              | u_c6fdkv9e4mvf26h42310 | github | github | 2021-12-10T19:16:54Z | connection | 1          | <null>     | <null>
| c_c6pqe28aa6gutha5g4c0 | o_c6pqdjoaa6gutha5g3tg | cody-org        | cody-org              | o_c6pqdjoaa6gutha5g3tg | aws    | aws    | 2021-12-10T19:14:17Z | connection | 1          | <null>     | {"access_key":"AKIA4B4LZUOLT4OO4N45","r
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------

> select * from steampipecloud_connection where identity_handle = 'cody-org' and handle = 'aws';
+----+-------------+-----------------+-----------------------+---------------+--------+--------+------------+------+------------+------------+--------+----------+
| id | identity_id | identity_handle | identity_display_name | identity_type | handle | plugin | created_at | type | version_id | updated_at | config | identity |
+----+-------------+-----------------+-----------------------+---------------+--------+--------+------------+------+------------+------------+--------+----------+
+----+-------------+-----------------+-----------------------+---------------+--------+--------+------------+------+------------+------------+--------+----------+
> select * from steampipecloud_connection where identity_handle = 'cody-org'
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
| id                     | identity_id            | identity_handle | identity_display_name | identity_type          | handle | plugin | created_at           | type       | version_id | updated_at | config
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
| c_c6pqe28aa6gutha5g4c0 | o_c6pqdjoaa6gutha5g3tg | cody-org        | cody-org              | o_c6pqdjoaa6gutha5g3tg | aws    | aws    | 2021-12-10T19:14:17Z | connection | 1          | <null>     | {"access_key":"AKIA4B4LZUOLT4OO4N45","r
+------------------------+------------------------+-----------------+-----------------------+------------------------+--------+--------+----------------------+------------+------------+------------+----------------------------------------
> select * from steampipecloud_connection where identity_handle = 'cody-org' and handle = 'aws';
+----+-------------+-----------------+-----------------------+---------------+--------+--------+------------+------+------------+------------+--------+----------+
| id | identity_id | identity_handle | identity_display_name | identity_type | handle | plugin | created_at | type | version_id | updated_at | config | identity |
+----+-------------+-----------------+-----------------------+---------------+--------+--------+------------+------+------------+------------+--------+----------+
+----+-------------+-----------------+-----------------------+---------------+--------+--------+------------+------+------------+------------+--------+----------+
```

Reply:
@cbruno10 It is working for me using STEAMPIPE_CLOUD_TOKEN - 
```
> select * from steampipecloud_connection
+------------------------+------------------------+-----------------+---------------+--------+--------+----------------------+------------+------------+------------+--------+------------------------------
| id                     | identity_id            | identity_handle | identity_type | handle | plugin | created_at           | type       | version_id | updated_at | config | identity                     
+------------------------+------------------------+-----------------+---------------+--------+--------+----------------------+------------+------------+------------+--------+------------------------------
| c_c6i87khe4mvahd3kq32g | u_c6g7fd9e4mv9br63ads0 | bigdatasourav   | user          | rss    | rss    | 2021-11-29T07:40:02Z | connection | 1          | <null>     | <null> | {"display_name":"souravTurbot
+------------------------+------------------------+-----------------+---------------+--------+--------+----------------------+------------+------------+------------+--------+------------------------------
> select * from steampipecloud_connection WHERE handle = 'rss' and identity_handle = 'bigdatasourav'
+------------------------+------------------------+-----------------+---------------+--------+--------+----------------------+------------+------------+------------+--------+------------------------------
| id                     | identity_id            | identity_handle | identity_type | handle | plugin | created_at           | type       | version_id | updated_at | config | identity                     
+------------------------+------------------------+-----------------+---------------+--------+--------+----------------------+------------+------------+------------+--------+------------------------------
| c_c6i87khe4mvahd3kq32g | u_c6g7fd9e4mv9br63ads0 | bigdatasourav   | user          | rss    | rss    | 2021-11-29T07:40:02Z | connection | 1          | <null>     | <null> | {"display_name":"souravTurbot
+------------------------+------------------------+-----------------+---------------+--------+--------+----------------------+------------+------------+------------+--------+------------------------------
> .cache off
> .cache clear
> select * from steampipecloud_connection WHERE handle = 'rss' and identity_handle = 'bigdatasourav'
+------------------------+------------------------+-----------------+---------------+--------+--------+----------------------+------------+------------+------------+--------+----------+
| id                     | identity_id            | identity_handle | identity_type | handle | plugin | created_at           | type       | version_id | updated_at | config | identity |
+------------------------+------------------------+-----------------+---------------+--------+--------+----------------------+------------+------------+------------+--------+----------+
| c_c6i87khe4mvahd3kq32g | u_c6g7fd9e4mv9br63ads0 | bigdatasourav   | user          | rss    | rss    | 2021-11-29T07:40:02Z | connection | 1          | <null>     | <null> | <null>   |
+------------------------+------------------------+-----------------+---------------+--------+--------+----------------------+------------+------------+------------+--------+----------+

```